Added a NNID warnings and removed outdated info on Juxt (#375)
PretendoNetwork/website: docs/en_US/install/3ds.md

# 3DS/2DS Family

> <div class="tip red">
> 	<strong>!!! DISCLAIMER !!!
> 	DON'T REMOVE THE NNID TO SET UP PRETENDO NETWORK, IT'S NOT NECESSARY AND BY DOING SO YOU WILL LOSE BADGES, REDOWNLOADABLE GAMES, THEMES AND POKÉMON BANK DATA!</strong>
> </div>

Reviewer: ```suggestion
<div class="tip red">
	<strong>CAUTION:</strong>
	DON'T REMOVE YOUR NNID TO SET UP PRETENDO NETWORK, IT'S NOT NECESSARY AND BY DOING SO YOU WILL LOSE BADGES, REDOWNLOADABLE GAMES, THEMES AND POKÉMON BANK DATA!
</div>
```
